editor/Scripting/ScriptManager.cs

```diff
@@ -181,49 +180,8 @@ private void updateSolutionFiles()
             if (!Directory.Exists(vsCodePath))
                 Directory.CreateDirectory(vsCodePath);
 
-            var vsCodeSettingsPath = Path.Combine(vsCodePath, "settings.json");
-            if (!File.Exists(vsCodeSettingsPath))
-                File.WriteAllBytes(vsCodeSettingsPath, resourceContainer.GetBytes("project/vscode_settings.json", ResourceSource.Embedded | ResourceSource.Relative));
-
```

Comment:
This code enforced a setting in VSC to tell OmniSharp, the engine behind the C# extension, that .NET framework is used. Since its now .NET 8, this is obsolete. But I'll remove the resource, forgot that.